src/reconstruction/modelGeneration/massBalance/checkMassChargeBalance.m

```diff
@@ -190,7 +190,7 @@
         end
     end
 
-    imBalancedCharge=model.S'*model.metCharges;
+    imBalancedCharge=model.S' * double(model.metCharges);
```

Comment:
Is there any reason for the conversion to `double`?

Reply:
This operation does not work on `int` arguments - this conversion was necessary to make it proceed

Reply:
May you put there a comment stating that this is necessary as `model.metCharges` might be stored as `int`? Thanks.

Reply:
Done